### PR TITLE
Add improved game information on finalize

### PR DESCRIPTION
**Pull request by TarikViehmann**
In order to make the process of submitting remote challenges easier, improve the refbox log output to contain all the relevant information.

Also remove some inconsistencies in the provided configs.

### CODE REVIEW

File: src/games/rcll/game.clp [hunk 1]
```diff
@@ -403,10 +403,51 @@
   (game-print-points-team MAGENTA)
 )
 
+(deffunction machine> (?m1-fact ?m2-fact)
```
Comment (snoato) [inline src/games/rcll/game.clp:406]: I didn't know CLIPS allows using `>` in function names. Nice ^^

File: src/refbox/main.cpp [hunk 2]
```diff
@@ -42,29 +42,17 @@
...
 #endif
 
-#include <termios.h>
-
 using namespace llsfrb;
 
 int
 main(int argc, char **argv)
 {
-	// Disable annoying display of ^C on Ctrl-C
```
Comment (snoato) [inline src/refbox/main.cpp:52]: What was the purpose of this originally and why was it removed?

Comment (TarikViehmann) [reply src/refbox/main.cpp:52]: I suppose exactly what the text states (not displaying ^C if you kill the refbox).
The reason why it is removed is, that if the refbox shuts down without restoring the termios, then this affects your running terminal that executes the refbox. In fedora34 this lead to you not being able to see what you type anymore. In Fedora 33 i did not encounter that issue...

**Review by snoato — APPROVED** [review]
Looks good! The new end-of-game outputs are really nice, thank you!

**Pull request merged**